tests/ui/const-generics/generic_const_exprs/expected-type-of-closure-body-to-be-a-closure-or-coroutine-ice-113776.stderr

```diff
@@ -62,7 +39,7 @@ help: you might have meant to write a bound here
 LL | ) -> impl Iterator<Item: SubAssign> {
    |                        ~
```

Comment:
Pre-existing, I feel that if we wait long enough to fix it, the syntax will be valid in stable 😈